{
  "comments": [
    {
      "key": {
        "uuid": "5ac10518_83cd9b5f",
        "filename": "/COMMIT_MSG",
        "patchSetId": 1
      },
      "lineNbr": 7,
      "author": {
        "id": 5959
      },
      "writtenOn": "2017-01-24T07:13:48Z",
      "side": 1,
      "message": "I would suggest to put more background information about the patch as there are two patches already trying to fix the error.",
      "range": {
        "startLine": 7,
        "startChar": 9,
        "endLine": 7,
        "endChar": 34
      },
      "revId": "f07f860e363b84a6abc6366c788ed09155c0071c",
      "serverId": "bbac25d2-bf60-4904-9ba8-a72fc000d6c5",
      "unresolved": false
    }
  ]
}